docs/team/cravetocode.md

```diff
@@ -8,19 +8,19 @@ title: CraveToCode's Project Portfolio Page
 #### Overview
 
 NurseyBook is a desktop application made for nurses in nursing homes to aid them in managing contacts and tasks in their busy lives. The user interacts with it using Command Line Interface (CLI),
-and it still has the benefits of a Graphical User Interface (GUI) created with JavaFX. It is written in Java, and has about 17 kLoC.
+and it still has the benefits of a Graphical User Interface (GUI) created with JavaFX. It is written in Java, and has about 19 kLoC.
 
 #### Summary of Contributions
 
-Given below are my contributions to the project.
+Given below are my key contributions to the project.
 
-* **New Feature**: Added the ability to view a preview of tasks that will be occurring on a future date.
+* **New Feature**: Added the `viewSchedule` command to allow a previewing of tasks that will be occurring on a future date
```

Comment:
Maybe you want to remove 'a'?

Reply:
Thanks for the notice, I will update it in the next PR!